tests/Model/EditableFormFieldTest.php

```diff
@@ -72,18 +72,13 @@ public function testCustomRules()
         // form has 2 fields - a checkbox and a text field
         // it has 1 rule - when ticked the checkbox hides the text field
         $this->assertEquals(1, $rules->Count());
-        $this->assertEquals($rules, $checkbox->EffectiveDisplayRules());
```

Comment:
Might need to restore these test along with restoring the `EffectiveDisplayRules`

Reply:
Done.